livekit-plugins/livekit-plugins-deepgram/livekit/plugins/deepgram/stt.py

```diff
@@ -303,6 +310,8 @@ async def _run(self, max_retry: int) -> None:
                     "filler_words": self._opts.filler_words,
                     "keywords": self._opts.keywords,
                     "profanity_filter": self._opts.profanity_filter,
+                    "base_url": self._base_url,
+                    "base_url_ws": self._base_url_ws,
```

Comment:
```suggestion
```

Reply:
it'd be better to pass these directly to `_to_deepgram_url` instead of sticking it into the live_config. the latter will send these unnecessary fields to deepgram in the URL.